config/routes.rb

```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
   root "home#index"
+  get '/auth/:provider/callback' => 'sessions#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.